index.rst

```diff
@@ -46,77 +46,233 @@
-
-    * Results
-        * This is were we got major agreement:
-          we want to store the tests results (including planning, observing, processing) in to technotes (just like this one!).
-        * Some tasks may not get to this stage, but only require a short summary of results in the JIRA ticket.
-
-Example JIRA ticket Workflow
-============================
-
-   - Stakeholder develops idea for AuxTel nighttime task.
-   - Task is proposed at AuxTel pre-planning meeting and adpoted
-   - Task is captured in a JIRA task ticket, which may/may not be associated with a top-level ticket for upcoming run/night.
-     Ticket is elevated to Proposed.
-   - A procedure/test script is developed such that it is ready for execution by the night crew.
-     Further development should not be required.
-   - JIRA ticket is reference in observation planning confluence page making it visible to nightcrew
-     (observation planning confluence should likewise be linked in JIRA ticket)
-   - Nightcrew executes task as described in JIRA ticket, capturing relevant information in nightlog.
-   - Daycrew extracts information from nightlog and populates JIRA ticket, linking nightlog.
-     Ticket is elevated to "Ready to Observe"
-   - Daycrew informs stakeholders/task proposers of ticket and elevates it to Analysis.
-   - Stakeholders/task proposers perform analysis and report status back to JIRA ticket or ticket reporter.
-     Ticket is elevated to Done.
+   This process will go through a major overhaul in the coming months, but this document acts as the central point for discussion.
+
+Introduction
+============
+
+The coordination of night time and daytime activities is continuing to increase as we continue to progress into the commissioning phase.
+This document outlines how Auxiliary Telescope observing runs are currently being conducted, from the observation planning stage, through the data reduction phase, and including the impacts on day crews providing critical troubleshooting of issues that have arisen during the night.
+
+Some of the overarching goals are:
+
+- Have night plans point to tasks with practiced and vetted procedures
+- Create a central repository for all information related to a task executed at night, which then follows to analyses when appropriate
+- A strategy to know how and when the tasks were run and how the data was processed.
+- Data analysis and results from the data need to be easily related to when the data was taken.
+- A searchable archive to and find tasks and their results.
+
+This document describes the state of the system as it is today, with the goal of making steady improvements, to which there will be many!
+This technote is designed to be a "living document" that is updated as the process evolves.
+
+Run Planning Blocks and Epics
+=============================
+
+At the time of writing, there are currently two runs AuxTel runs per month, each consisting of 3 nights.
```

Comment:
"there are currently two runs AuxTel runs per month"

->

"there are currently two AuxTel runs per month"


each consisting of 3 nights. -> each consisting of 3 consecutive nights.